plugins/config_manager.py

```diff
@@ -88,6 +91,9 @@ def modify_repo(self):
         modify = {}
         if hasattr(self.opts, 'main_setopts') and self.opts.main_setopts:
             modify = dict(self.opts.main_setopts._get_kwargs())
+        if self.opts.dump_variables:
+            for name, val in self.base.conf.substitutions.items():
+                print("%s = %s" % (name, val))
```

Comment:
then I think quoting should be added in config dump as well.... but it's for another PR.